cloud/client.go

```diff
@@ -408,7 +565,17 @@ func (c *client) signChallenge(challenge string, key *agent.Key) (string, error)
 	return s, nil
 }
 
-func (c *client) tryAuth(challenge string, key *agent.Key) (string, string, error) {
+func (c *client) getSSHCredentials(challenge string, key *agent.Key) (credentials string, err error) {
```

Comment:
Can this be refactored so we avoid duplicating the credential string formatting?

For example, we now have `fmt.Sprintf("ssh-rsa %s %s", blob, sig)` showing up twice in our code-base, where as previously we only had it in a single location.

Reply:
Good idea 👍

Reply:
actually, the other place was unused code so it's been removed.